lib/rest2.js

```diff
@@ -98,6 +103,24 @@ class RESTv2 {
     this._transform = !!opts.transform
     this._agent = opts.agent
     this._affCode = opts.affCode
+    this._timeout = opts.timeout || BASE_TIMEOUT
```

Comment:
this won't allow 0 value

Reply:
Are you sure that we want to allow it? Can add, but maybe would be better to allow users to set a big timeout instead of 0

Reply:
well it's up to user if he wants a timeout or not so I believe that we should allow that

Reply:
okay)